src/app/chat/view.nim

```diff
@@ -845,4 +848,11 @@ QtObject:
       self.status.chat.removeUserFromCommunity(self.activeCommunity.id(), pubKey)
       self.activeCommunity.removeMember(pubKey)
     except Exception as e:
-      error "Error removing user from the community", msg = e.msg
\ No newline at end of file
+      error "Error removing user from the community", msg = e.msg
+
+  proc generateIdenticon*(self: ChatsView, pk: string): string {.slot.} =
```

Comment:
`utilsView` already has this function. Maybe `generateAlias` could be added to it as well?

Reply:
Yea there is, but this particular component isn't related to the `utilsView` so I thought I'll replicate that API here. Or is `utilsView` indeed "service" view that is supposed to be reused in different places?

Reply:
`utilsView` is really just Utils meant to be used anywhere in in the app